package.json

```diff
@@ -164,7 +164,7 @@
     "json-dup-key-validator": "1.0.3",
     "json-stringify-pretty-compact": "3.0.0",
     "json5": "2.2.0",
-    "luxon": "1.28.0",
+    "luxon": "2.0.1",
```

Comment:
All the test are passing, hooray! 🎉 ✨ 

Important quotes from the upgrade guide at https://moment.github.io/luxon/#/upgrading

> **Environment support**
>
> Luxon 2.0 does not support Node < 12, or any version of IE. It also only supports newer versions of major browsers. This change allows Luxon to make more assumptions about what's supported in the environment and will allow Luxon's code to simplify. See the [Support Matrix](https://moment.github.io/luxon/#/matrix) for more.
>
> For this same reason, a polyfilled build is no longer provided; everything Luxon needs comes standard on browsers.

I don't know if these changes make the upgrade to Luxon 2 breaking for our end-users. Maybe this update should land in a new major version of Renovate?

---

> **Breaking signature changes**
>
> There are many more specific breaking changes. Most are aimed and making Luxon's handling of option parameters more consistent.

The migration guide shows how you can refactor the existing code to use the new option parameter style. I can make a new issue after we merge this PR to track this refactoring effort.